cmd/webhooks/main.go

```diff
@@ -74,14 +77,37 @@ func main() {
 	cfg := ctrl.GetConfigOrDie()
 	cfg.QPS = webhooksClientRequestQPS
 	cfg.Burst = webhooksClientRequestBurst
+	// Configuring minimum TLS version for the webhook server
+	var minTLSVersion uint16
+
+	switch tlsMinVersion {
+	case "1.0":
+		minTLSVersion = tls.VersionTLS10
+	case "1.1":
+		minTLSVersion = tls.VersionTLS11
+	case "1.2":
+		minTLSVersion = tls.VersionTLS11
+	case "1.3":
+		minTLSVersion = tls.VersionTLS13
+	default:
+		setupLog.Error(fmt.Errorf("unsupported minimum TLS version"), fmt.Sprintf("option %s non recognized", tlsMinVersion))
```

Comment:
would be nice to put this into https://github.com/kedacore/keda/blob/main/pkg/util/tls_config.go

Reply:
We have it indeed, or at least something similar: https://github.com/kedacore/keda/blob/d917094cadbb5fa132e2530c70a5ddfd94710471/pkg/util/tls_config.go#L89-L108

Reply:
And there is another function to query the value wherever is needed:
https://github.com/kedacore/keda/blob/d917094cadbb5fa132e2530c70a5ddfd94710471/pkg/util/tls_config.go#L85-L87

Reply:
`GetMinTLSVersion` is not taking into consideration the input value from the user.

I just added a refactoring commit promoting the function `initMinTLSVersion` to public visibility by renaming it into `ParseTLSMinVersionAsString`.

Reply:
I'd say more... Taking into account that tls-min-version isn't documented anywhere, I'd remove it directly as part of this commit in favor of the documented env.
@zroubalik @tomkerkhove ?
Just to clarify, that parameter was introduced in v2.10 and is not documented anywhere, so if anyone is using it is because they have reviewed the code and used it under their own risk.

Reply:
I committed the suggested change with an atomic commit (e70fd67f) to make it easier to drop or keep the changes.